packages/edit-site/src/components/page-pages/index.js

```diff
@@ -26,13 +26,23 @@ const EMPTY_OBJECT = {};
 export default function PagePages() {
 	const [ view, setView ] = useState( {
 		type: 'list',
-		search: '',
+		filters: {
+			search: '',
+			status: 'publish, draft',
```

Comment:
I think for enumerations we are going to support "is" filters and "is not" filters. Should our structure take that into account and use something like:
```
status: { is: 'publish, draft' }
```
or 
```
status: 'is: publish, draft' 
```

Reply:
I'd like to implement a `NOT IN` filter in a follow-up to showcase how it can work. In a previous exploration (filters in columns), I did it by declaring this metadata in the field:

```
filters: [
  { id: 'author', type: 'in' },
  { id: 'author_exclude', type: 'notIN' },
]
```

I think something like that is doable for top-level filters as well.

Reply:
I think at the UI level a single component will render the in and not in filter. That seems to be what happens in current applications. 
So I don't think we should have this shape:
```
filters: [
  { id: 'author', type: 'in' },
  { id: 'author_exclude', type: 'notIN' },
]
```

Because in and notIn are a single type of filter that renders a UI that will allow to pick an author part of the set or exclude an author from the set. Rendering two components separately will easily pollute the UI.

The same for text filters, if a field is of type text its filter will allow in a single component things like contains, starts with etc.

Reply:
I went with your suggestion `type: enumeration`. I think that could be enough as well, though, there's a subtle difference between `IN` and `NOT IN` filters: the label for the reset state. For `IN` filters we want `All` and for `NOT IN` filters we want `None`. We could also find a string that works for both.

https://github.com/WordPress/gutenberg/pull/55440 changes a bit how the "reset option" works: having one by default but allowing filters to change it, should they need. This would help to provide a different string for `NOT IN` filters, for example, while we maintain the filter type as `enumeration`.

Reply:
https://github.com/WordPress/gutenberg/pull/55508 tries to bring back the column filters exploration, so revisits the idea of more granularity for filter types (see [comment](https://github.com/WordPress/gutenberg/pull/55508#discussion_r1366827641) about `enumeration` vs `enumeration_in`/`enumeration_not_in`).